scripts/generate_libp2p.sh

```diff
@@ -9,22 +9,36 @@
 # - Docker: https://docs.docker.com/engine/install
 # - Mina Daemon: https://docs.minaprotocol.com/node-operators/getting-started
 
+# Usage: ./generate_libp2p.sh [docker|binary]
+# - docker: Generate keypair using Docker only
+# - binary: Generate libp2p keypair using mina binary only
 
 # Enable debug mode
 set -x
 
 # Exit on error, and pipefail
 set -eo pipefail
+. ../.env
+
+# Get mode from first argument
+MODE="${1}"
+
+# Validate mode
+if [[ "$MODE" != "docker" && "$MODE" != "binary" ]]; then
+  echo "Usage: $0 [docker|binary]"
+  echo "  docker - Generate keypair using Docker only"
+  echo "  binary - Generate libp2p keypair using mina binary only"
+  exit 1
+fi
 
 # Set environment variable
 # Make sure these match the environment variables used in the docker-compose.yml file
-export MINA_PRIVKEY_PASS="passlib"
-export MINA_LIBP2P_PASS="passlib"
+export MINA_LIBP2P_PASS=$MINA_LIBP2P_PASS
 
 # Constants
 KEYPAIR_DIR="keys"
 KEYPAIR_NAME="libp2p-keys"
-MINA_KEYPAIR_IMAGE="minaprotocol/mina-generate-keypair:1.3.0-9b0369c"
+MINA_KEYPAIR_IMAGE=$MINA
```

Comment:
Nit: we no longer need this package, I think. mina's main docker has this functionality bundled.